geth/api/backend_test.go

```diff
@@ -173,6 +173,85 @@ func (s *BackendTestSuite) TestStartNodeWithUpstreamEnabled() {
 
 	<-nodeStarted
 	require.True(backend.IsNodeRunning())
+
+	// Get lightEthereum service and attempt to download sync.
+	lightEth, err := backend.NodeManager().LightEthereumService()
+	require.NoError(err)
+	require.NotNil(lightEth)
+	time.Sleep(5 * time.Second)
+	require.False(lightEth.Downloader().Synchronising(), "NodeChain synchronization is not started")
```

Comment:
I guess the commend should describe the problem so `"NodeChain synchronization should've not started"`?